index.md

```diff
@@ -82,7 +88,7 @@ We organize the conference using Trello:
 
 Have you ever been coding under the northern lights? Or in a Lapp daylight? Early-mid February is one of the best times to see northern lights in action. And it also happens, that there's a full moon during the stay. We've done what we can - provide best possible time for some extraordinary experiences.
 
-We know it might be rather cold. Temperatures down to -25 ºC are normal at the time of the year. For some clothing tips, visit [here](faq).
+We know it might be rather cold. Temperatures down to -25 ºC are normal at the time of the year. For some clothing tips, see the [FAQ](#faq).
```

Comment:
```suggestion
We know it might be rather cold. Temperatures down to -25°C are normal at the time of the year. For some clothing tips, see the [FAQ](#faq).
```